web/src/engine/predictive-text/worker-thread/src/tests/mocha/cases/suggestion-similarity.js

```diff
@@ -340,8 +340,8 @@ describe('processSimilarity', () => {
       prediction: {
         sample: {
           transform: {
-            insert: 'e',
-            deleteLeft: 0
+            insert: 'iphone',
+            deleteLeft: 5
           },
```

Comment:
Note:  this unit test applies at a lower level - one that doesn't apply post-word formatting.